frontend/packages/ceph-storage-plugin/locales/en/dashboard.json

```diff
@@ -0,0 +1,3 @@
+{
+  "Not available": "Not available"
```

Comment:
I'm unsure why we have a new namespace in ceph-storage-plugin. @zherman0 Was this created on purpose?

Reply:
It was on purpose... once I removed the dashboard.json from public/locales/en, the i18n auto generated that file to the /package/ceph/locales.  The complete solution would be to go find where that is used in the ceph code and change it to ceph-storage.json but I decided to leave that out of this scope since it involves 5 ceph files.  Let me know if you want me to make those changes as part of this PR.

Reply:
@zherman0 Can you open a bug against the storage plugin component to make this change?

Reply:
This has been resolved by PR 8764.